docs/backend/index.md

```diff
@@ -22,6 +22,7 @@ behaviors
 configuration-registry
 content-types/index
 control-panels
+deprecation
```

Comment:
Do you all think that this insertion point in the TOC is the most appropiate place for this subject?

Reply:
It is not. See https://github.com/plone/documentation/pull/1850#pullrequestreview-2614132745.